pkg/cluster/update.go

```diff
@@ -90,7 +90,7 @@ func Update(
 	}
 	rootApp, err := ConstructRootApp(argocdNs, clusterName, "", repoUrl,
 		repoBranch, repoPath, clusterSpecName, clusterSpecCm, prof.Name,
-		managementClusterUrl)
+		managementClusterUrl, false)
```

Comment:
Is Update supported for gen2 clusters (it's been a while since I've touched this)? And if so, is there a risk of turning off CAS here if it was previously set.

Reply:
Updates for gen2 happen via `ngupdate` not `update`